osbrain/agent.py

```diff
@@ -1772,7 +1783,7 @@ def run_agent(name, nsaddr=None, addr=None, base=Agent, serializer=None,
 
     Parameters
     ----------
-    name : str
+    name : str, default is None
```

Comment:
Default is `''`. :wink:

We might remove all this redundancy when we integrate typing. For now we will keep it for style consistency (this is the NumPy/Pandas/Scipy style, which we initially chose).